### DISCUSSION TITLE
How to do development on your local machine for a subdomain with auth?

### DISCUSSION THREAD

**Discussion by progrmoiz** [dubinc/dub#166] [Q&A]
For example, logging in on localhost:3000 and keeping that session for app.localhost:3000. 

On Vercel it seems to work fine, like with an actual domain, but on your local machine with localhost:3000 it only stores it for localhost:3000.

**Comment by steven-tey** [comment]
We recently switched to `localhost:8888` for the app (`app.dub.co`) and `home.localhost:8888` for the landing page (`dub.co`) for this very reason, since Google auth doesn't allow for localhost subdomains
